go/signedexchange/README.md

```diff
@@ -1,3 +1,83 @@
-# gen-signedexchange
+# go/bundle
+This directory contains a reference implementation of [Signed HTTP Exchanges](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html) spec.
```

Comment:
I believe it's only a reference implementation to generate signed exchanges, rather than the whole spec?

Reply:
We plan to add the envelope parser later, but yes. Corrected.